sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -172,8 +172,6 @@ rename-mapping:
   VMDiskSecurityProfile: VirtualMachineDiskSecurityProfile
   VmDiskTypes: VirtualMachineDiskType
   VMGalleryApplication: VirtualMachineGalleryApplication
-  VMGuestPatchClassificationLinux: LinuxVmGuestPatchClassification
-  VMGuestPatchClassificationWindows: WindowsVmGuestPatchClassification
```

Comment:
We should not remove these two, or we need to add the later to the list of preserve plural enum names - because the windows suffix is changed to window here which makes it no longer make sense